lib/rdoc/task.rb

```diff
@@ -248,6 +251,16 @@ def define
       RDoc::RDoc.new.document args
     end
 
+    namespace rdoc_task_name do
+      desc coverage_task_description
+      task coverage_task_name do
+        @before_running_rdoc.call if @before_running_rdoc
+        opts = option_list << "-C"
+        args = opts + @rdoc_files
+
+        $stderr.puts "rdoc #{args.join ' '}" if Rake.application.options.trace
+      end
+    end
```

Comment:
```suggestion
    end

```

I think the earlier one had a newline before returning `self`.